.pylintrc

```diff
@@ -333,7 +333,7 @@ indent-string='    '
 max-line-length=100
 
 # Maximum number of lines in a module.
-max-module-lines=1200
+max-module-lines=1400
```

Comment:
Seems like we will keep hitting this. I also had to `disable` this specifically for `test_sft_trainer.py`

https://github.com/foundation-model-stack/fms-hf-tuning/pull/409/files#diff-406c3c4d02c1ff158009b04d26a6b2d357f05d11e8c35c3f2ddfc1b54649a022R18

Reply:
Sounds good. Would take in this change after your PR is merged.